core/web/auth/auth.go

```diff
@@ -78,12 +79,12 @@ func AuthenticateByToken(c *gin.Context, authr Authenticator) error {
 		Secret:    c.GetHeader(APISecret),
```

Comment:
Hmm, I think it would be better if we cut a ticket and address that separately for the scope here - we can include it for now and evaluate with other stakeholders?

Reply:
Great! I think we should do that ticket first, then merge this one on top

Reply:
I'd like to hold off on purging token auth as we were able to slot in review time for it including the scope of the change here, it should be straightforward to remove in a future PR